### ISSUE TITLE
refresh drawer content

### ISSUE THREAD

**Issue by edelworksgithub** [root-two/react-native-drawer#200]
Is there a way to refresh/re-render/reload the drawer content, while drawer is opened

**Comment by suckkay** [comment]
yeah i need this function also, because my translation wont change until i refresh drawer

**Comment by Cody-G-G** [comment]
You can have some state that you use in the drawer render method....when that state is changed it will trigger a re-render just like for any other component. I'm having problems in release mode with it though, it re-renders but the UI doesn't actually update for some reason....

**Comment by carlo-fontanos** [comment]
Yep, state is the way to go.

**Comment by mymys91** [comment]
Could you suggest which method/event is called when the drawer content opens? I need it to set state
